marked-yaml/Cargo.toml

```diff
@@ -16,7 +16,5 @@ maintenance = { status = "experimental" }
 
 [dependencies]
 doc-comment = "0.3"
-# We lock the yaml-rust version down hard for now
-yaml-rust = "=0.4.5"
-# We should keep this in sync with the use from yaml-rust
-linked-hash-map = "=0.5.3"
+yaml-rust = "0.4.5"
+linked-hash-map = "0.5.6"
```

Comment:
This lockdown was meant to be so that we didn't end up with different versions of linked-hash-map in two places resulting in type problems - Is this no longer an issue?

Reply:
Looking at the lockfile it seems to resolve only to a single `linked-hash-map` version, so I do think we're fine here.

Reply:
PS: We've been using this patch for quite some time in `rattler-build` now and it works fine :)

Reply:
👍🏻